src/control/server/config/faults.go

```diff
@@ -216,13 +226,42 @@ func FaultConfigNrHugepagesOutOfRange(req, max int) *fault.Fault {
 	)
 }
 
-// FaultConfigInsufficientHugePages creates a fault for the scenario where the
+// FaultConfigInsufficientHugepages creates a fault for the scenario where the
 // number of configured huge pages is less than required.
-func FaultConfigInsufficientHugePages(min, req int) *fault.Fault {
+func FaultConfigInsufficientHugepages(min, req int) *fault.Fault {
+	return serverConfigFault(
+		code.ServerConfigInsufficientHugepages,
+		fmt.Sprintf("insufficient huge pages configured for the number of targets (%d < %d)",
+			req, min),
+		"remove the 'nr_hugepages' parameter so it can be automatically set or update "+
+			"it to a higher value in the config file",
+	)
+}
+
+// FaultConfigScmTmpfsUnderMinMem indicates that the tmpfs size requested in config is less than
+// minimum allowed.
+func FaultScmTmpfsUnderMinMem(confSize, scmSize, memTmpfsMin uint64) *fault.Fault {
+	return serverConfigFault(
+		code.ServerConfigScmTmpfsUnderMinMem,
+		fmt.Sprintf("configured scm tmpfs size %s is lower than the minimum (%s) required "+
+			"for SCM", humanize.IBytes(confSize), humanize.IBytes(memTmpfsMin)),
+		fmt.Sprintf("remove the 'scm_size' parameter so it can be automatically set "+
+			"or manually set to a value between %s and %s in the config file",
+			humanize.IBytes(memTmpfsMin), humanize.IBytes(scmSize)),
+	)
+}
+
+// FaultConfigScmTmpfsOverMaxMem indicates that the tmpfs size requested in config is larger than
+// maximum allowed.
+func FaultScmTmpfsOverMaxMem(confSize, scmSize, memTmpfsMin uint64) *fault.Fault {
 	return serverConfigFault(
-		code.ServerConfigInsufficientHugePages,
-		fmt.Sprintf("insufficient huge pages configured for the number of targets (%d < %d)", req, min),
-		"update the 'nr_hugepages' parameter or remove it for automatic configuration",
+		code.ServerConfigScmTmpfsOverMaxMem,
+		fmt.Sprintf("configured scm tmpfs size %s is larger than the maximum (%s) that "+
+			"total system memory (RAM) will allow", humanize.IBytes(confSize),
+			humanize.IBytes(scmSize)),
+		fmt.Sprintf("remove the 'scm_size' parameter so it can be automatically set "+
```

Comment:
Thanks @knard-intel . Could you please elaborate as to why you think it's not suited? The resolution is similar because removing the parameter from the configuration will result in a correct value being calculated.

Reply:
My apologies, the message is perfectly fine. I was confused by the scmSize wich is the maximal size.